components/kubeflow/pytorch-launcher/requirements.txt

```diff
@@ -1,4 +1,6 @@
 pyyaml
 kubernetes
 kubeflow-pytorchjob
+kubeflow.training
```

Comment:
Is there minimal required version?

Reply:
Thank you. I've updated the requirements.txt to include more specific dependencies - see this [commit](https://github.com/kubeflow/pipelines/pull/11273/commits/6d980bd317b5b8285c3a2d4bd83c02983c8ae993)